docs/developing-contracts/examples/erc-721.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_label: 'ERC-721: NFT'
+sidebar_position: 3
+---
+# Introduction
+
+At Gear, we want to provide an ecosystem for developers coming from various backgrounds. We have to note that Gear offers far more advanced features and technological developments compared to platforms that popular protocols like ERC-20 or ERC-721 were designed for. However, it would be inefficient to ignore such widely used interfaces in favor of arbitrary, but modern implementations. Therefore, Gear provides support for ERC-721 based NFT tokens, as well as an updated standard based on Gear's vision.
+
+In this article, we will cover usage of Gear's ERC-721 interface library in composition with an example NFT implementation.
+
+# ERC-721 interface
+
+ERC-721 protocol is a community-accepted standard for Non-Fungible Tokens (NFTs) smart contract implementation. In this article, we will go over the details of how to use the ERC-721 interface provided by Gear via an example.
+
+[ERC-721](https://eips.ethereum.org/EIPS/eip-721) describes the interface that has to be implemented by a smart contract in order to be compliant. The following functions are expected to be present in the NFT contract:
+
+```rust
+/// This emits when ownership of any NFT changes by any mechanism.
+Transfer(from, to, tokenId);
+
+/// This emits when the approved address for an NFT is changed or reaffirmed.
+ApprovalForAll(_owner, operator, approved);
+
+/// Count all NFTs assigned to an owner
+balanceOf(owner): integer;
+
+/// Find the owner of an NFT
+ownerOf(tokenId): address;
+
+/// Transfers the ownership of an NFT from one address to another address
+safeTransferFrom(from, to, tokenId, data);
+
+/// Transfers the ownership of an NFT from one address to another address
```

Comment:
looks like it duplicates the previous one.

Reply:
Addressed